www/include/reporting/dashboard/xmlInformations/GetXmlService.php

```diff
@@ -61,8 +70,8 @@
     if ($accessService) {
         $DBRESULT = $pearDBO->query(
             "SELECT  * FROM `log_archive_service` WHERE host_id = '".
-            $pearDBO->escape($_GET["host_id"])."' AND service_id = ".
-            $pearDBO->escape($_GET["id"])." ORDER BY `date_start` DESC"
+            $pearDBO->escape($_GET["host_id"])."' AND service_id = '".
```

Comment:
Is it a good idea to update this query like that ? Because it's not working with mysql strict mode